css_tasks/task11/js/constants.js

```diff
@@ -0,0 +1,17 @@
+export const constants = {
+  url: '../data.json',
+  months: {
```

Comment:
month id is equal to index in array, so you can have just array here, what would be cleaner